packages/content-conformance/src/cli.ts

```diff
@@ -94,5 +100,7 @@ yargs(hideBin(process.argv)).command(
       console.log('')
       console.log(chalk.redBright(stack))
     }
+
+    process.exitCode = RunnerExitCode[runner.status ?? 'SUCCESS']
```

Comment:
I'm wondering if we can scope this to only the failure case, given that `0` is the default:
```suggestion
		if (runner.status === 'SUCCESS') {
			process.exitCode = 1
		}
```

Whatcha think?

Reply:
Looks good! I pushed a change